src/widget/core.rs

```diff
@@ -66,54 +70,49 @@ pub struct Pod {
     fragment: SceneFragment,
 }
 
-#[derive(Default, Debug)]
+#[derive(Debug)]
 pub(crate) struct WidgetState {
+    pub(crate) id: Id,
     pub(crate) flags: PodFlags,
+    /// The origin of the child in the parent's coordinate space.
     pub(crate) origin: Point,
+    /// The origin of the parent in the window coordinate space.
+    pub(crate) parent_window_origin: Point,
     /// The minimum intrinsic size of the widget.
     pub(crate) min_size: Size,
     /// The maximum intrinsic size of the widget.
     pub(crate) max_size: Size,
```

Comment:
The values of `min_size` and `max_size` are never changed. Are they remains of the exploration of `SwiftUI` layout?

Reply:
I will remove them, thanks for the heads-up.